src/modules/external/aha/components/AhaScore.tsx

```diff
@@ -83,16 +95,25 @@ const AhaScore = ({
     },
   });
 
-  // console.error instead of throwing an error, so we can still preview how the form looks in non-client contexts
+  // Throw an error if the children don't match the expected structure
+  if (!isComponentValid) throw new Error('Invalid Aha form component');
+
+  if (viewOnly) {
+    return item.item?.map((i) => {
+      if (i.linkId === SCORE_LINK_ID && !hasScore) {
+        return renderChildItem(i, { value: null });
```

Comment:
Actually saving the value as -1 in the db requires manually overwriting the display value so we show "Data not collected" instead of -1 when the assessment is reopened. To do that (and satisfy typescript) I've added the change in `types.ts` below, which definitely feels smelly.

Would it be ok to just display -1?